presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/base/ShowImageDialogFragment.java

```diff
@@ -0,0 +1,64 @@
+package com.globant.equattrocchio.cleanarchitecture.mvp.view.base;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.globant.equattrocchio.cleanarchitecture.R;
+import com.globant.equattrocchio.cleanarchitecture.mvp.presenter.ShowImagePresenter;
+import com.globant.equattrocchio.cleanarchitecture.mvp.view.ShowImageView;
+import com.globant.equattrocchio.data.GetImageServicesImpl;
+import com.globant.equattrocchio.domain.GetImageUseCase;
+import com.globant.equattrocchio.domain.response.Image;
+
+public class ShowImageDialogFragment extends DialogFragment {
+    public static Intent getStartIntent(Context context, Image image) {
+        Intent intent = new Intent(context, ShowImageDialogFragment.class);
+        intent.putExtra(EXTRA_IMAGE, image);
+        return intent;
+    }
+
+    public static ShowImageDialogFragment newInstance(Integer id) {
+        ShowImageDialogFragment fragment = new ShowImageDialogFragment();
+        Bundle args = new Bundle();
+        args.putInt(ARGUMENT_IMAGE, id);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    private static final String EXTRA_IMAGE = "extra_image";
```

Comment:
Move all this variables sections at the begining of the class. Leave a blank line bewteen the class theclaration and this section.